test/model.jl

```diff
@@ -268,8 +268,15 @@ function test_bridges_automatic_disabled()
     @test JuMP.backend(model) isa MOIU.CachingOptimizer
     @test !(JuMP.backend(model).optimizer isa MOI.Bridges.LazyBridgeOptimizer)
     @variable model x
+    F = MOI.ScalarAffineFunction{Float64}
+    S = MOI.Interval{Float64}
     err = ErrorException(
-        "Constraints of type MathOptInterface.ScalarAffineFunction{Float64}-in-MathOptInterface.Interval{Float64} are not supported by the solver, try using `bridge_constraints=true` in the `JuMP.Model` constructor if you believe the constraint can be reformulated to constraints supported by the solver.",
+        "Constraints of type $(F)-in-$(S) are not supported by the " *
+        "solver.\n\nIf you expected the solver to support your problem, " *
+        "you may have an error in our formulation. Otherwise, consider " *
+        "using a different solver.\n\nThe list of available solvers, " *
```

Comment:
Maybe say that the list is not exhaustive. Here it seems like if the solver is not in the list then it's not ecpected to work

Reply:
By the way, we should have a stable url like `jump.dev/solvers` as we reference to this so often it would be painful tonupdate if the link changes

Reply:
If people are using a solver that isn't in the list, we should add it to the list.

And agreed on the link, so we should just not change it!

Reply:
We say here: https://jump.dev/pages/code/
that it's not exhaustive though